bpf/flows.c

```diff
@@ -63,10 +63,18 @@ static inline void update_existing_flow(flow_metrics *aggregate_flow, pkt_info *
     }
     aggregate_flow->flags |= pkt->flags;
     aggregate_flow->dscp = pkt->dscp;
-    aggregate_flow->dns_record.id = pkt->dns_id;
-    aggregate_flow->dns_record.flags = pkt->dns_flags;
-    aggregate_flow->dns_record.latency = pkt->dns_latency;
-    aggregate_flow->dns_record.errno = dns_errno;
+    bpf_spin_unlock(&aggregate_flow->lock);
```

Comment:
we no longer needs https://github.com/netobserv/netobserv-ebpf-agent/blob/119061c05dbafe2dc07e45bd34419d3e58be23c2/bpf/flows.c#L59
as that was specific to perCPU map

Reply:
correct, good catch

Reply:
fixed